package.json

```diff
@@ -2,24 +2,21 @@
   "name": "apollo-server-monorepo",
   "private": true,
   "license": "MIT",
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/apollographql/apollo-server.git"
-  },
+  "repository": "github:apollographql/apollo-server",
   "scripts": {
-    "postinstall": "lerna run prepare",
+    "clean": "git clean -dfqX **/{dist,node_modules}/",
```

Comment:
I haven't actually tested it on Windows, but I knew it would be 100% more likely to work.  We might want to transplant this to Apollo Client for accessibility to Windows contributors!